### ISSUE TITLE
Add onTransitionEnd callback

### ISSUE THREAD

**Issue by a-koka** [oblador/react-native-animatable#85]
The `onAnimationEnd` callback doesn't seem to be firing for the following snippet. I'm using version `1.1.0`.

Wondering if this is a bug or am I doing something wrong?

```
<Animatable.Image
duration={1500}
transition={['width', 'height', 'borderRadius', 'borderWidth', 'marginTop']}
onAnimationEnd={() => alert('done')}
style={[styles.logo, {width: size, height: size, borderRadius: size/2, borderWidth, marginTop}]}
source={require('logo.png')} />
```

**Comment by hiyahin** [comment]
Checking the source in `createAnimatableComponent.js`, Line 237 and 253 says that `onAnimationEnd` only fires when using the `animation` prop. Maybe we can make an `onTransitionEnd` or make `onAnimationEnd` fire for transitions as well.

**Comment by a-koka** [comment]
Thanks for checking the source @hiyahin. Yes, an `onTransitionEnd` or a generic `onAnimationEnd` would be quite useful.

**Comment by vingrad** [comment]
+1
any news?

**Comment by neuling** [comment]
+1

**Comment by ragamufin** [comment]
+1

**Comment by tuantle** [comment]
+1

**Comment by marceloogeda** [comment]
+1

**Comment by javier-artiles** [comment]
+1

**Comment by redgenie** [comment]
This will really be helpful

**Comment by oblador** [comment]
`onTransitionBegin` and `onTransitionEnd` are now available in [1.3.0](https://github.com/oblador/react-native-animatable/releases/tag/v1.3.0).
